### DISCUSSION TITLE
Can't seem to get global variables working...

### DISCUSSION THREAD

**Discussion by jonathanthompson-me** [EdgeTX/edgetx#5405] [Q&A] [answered]
What are the requirements to get a global variable as an option for weight? RM MT12. Recent main branch compilation, a couple days old.
![20240807_212334](https://github.com/user-attachments/assets/e3af9711-6c99-4bf9-b3b9-2581434d7e83)
![20240807_212354](https://github.com/user-attachments/assets/c2a93334-183d-4645-9961-d690e783beaf)
![20240807_212415](https://github.com/user-attachments/assets/8030539c-9e93-4be8-be02-fc2ecc10b2bd)
![20240807_212424](https://github.com/user-attachments/assets/f1a991ba-5fe1-41ac-a0b4-5adb7a2610e4)

**Comment by pfeerick** [comment, marked as answer]
You probably want today's nightly so you have the fixes from https://github.com/EdgeTX/edgetx/pull/5386. In the popup list from your last screenshot, when you scroll further down the list, there will be a "GVars" entry.

**Comment by jonathanthompson-me** [comment]
That got it. Thanks!
